### PR TITLE
Fixed meme command

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/commands/fun/meme.js [hunk 1]
```diff
@@ -117,30 +117,29 @@ async function getRandomEmbed(choice) {
   const subReddits = ["meme", "Memes_Of_The_Dank", "memes", "dankmemes"];
   let rand = choice ? choice : subReddits[getRandomInt(subReddits.length)];
```
Comment (imnaiyar) [inline src/commands/fun/meme.js]: change this to
```javascript
let rand = choice ? choice : subReddits[getRandomInt(subReddits.length - 1)]
``` 
Array length starts from `1` but iterating an array starts from `0`,  so if the randomInt is by any chance equal to the array's length, it'll return undefined as number 4 would be for the 5th element in the array which doesn't exist here.

This issue was first pointed by @rhaym-tech

**Review by imnaiyar — CHANGES_REQUESTED** [review]

**Review by rhaym-tech — APPROVED** [review]
Good job, this is amazing!

**Review by imnaiyar — APPROVED** [review]
All good 😊😊! Thank you.

**Comment by saiteja-madha** [comment]
Works like a charm. Will be added to 5.6.0 release

**Pull request merged**